polynomial/polynomial_client.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+import logging
+from random import randint
+import grpc
+import polynomial_pb2
+import polynomial_pb2_grpc
+import polynomial
+import matplotlib.pyplot as plt
+import numpy as np
+value=[-1.,-0.77777778,-0.55555556, -0.33333333,-0.11111111,0.11111111,0.33333, 0.55555556, 0.77777778 , 1]
```

Comment:
don't use global variable if it's not a must. Just in case they are in polynomial coefficient forms, please put it in another file